apps/demo/vite.config.ts

```diff
@@ -9,15 +9,15 @@ export default defineConfig((_env) => {
     server: {
       proxy: {
         "/planner/api": {
-          target: "http://172.17.0.3:3443",
+          target: "http://127.0.0.1:3443",
           changeOrigin: true,
           rewrite: (path): string => path.replace(/^\/planner/, ""),
         },
-        "/agent/api/v1": {
-          target: "http://172.17.0.3:3333",
-          changeOrigin: true,
-          rewrite: (path): string => path.replace(/^\/agent/, ""),
-        },
+       // "/agent/api/v1": {
```

Comment:
Please revert those changes. It's prepared to work with dev-containers. I would not change it.